supermarq-benchmarks/supermarq/run_benchmarks.py

```diff
@@ -44,29 +41,25 @@ def get_qpu_targets(targets: Dict[str, List[str]]) -> List[str]:
         date = datetime.today().strftime("%Y-%m-%d")
         tag = f"{date}-{label}-{target}"
 
-        # Since there is no way to determine if a target supports mid-circuit measurement purely
-        # through `service.target_info()`, we use a `try` block to catch when we submit an invalid
-        # circuit to a device that does not support it.
-        # Related issue: https://github.com/Infleqtion/client-superstaq/issues/550.
+        target_info = service.target_info(target)
+        if label == "bitcode3" and not target_info.get("supports_midcircuit_measurement", False):
+            continue
 
-        try:
```

Comment:
should we still wrap this in a `try..except` block, so if it fails for a target because e.g. some provider is down the rest of the benchmarks would still run?

Reply:
ah yes, that makes sense, will add that back in.